src/card/Card.js

```diff
@@ -80,9 +81,10 @@ const Card = props => {
 
         {image && (
           <View style={imageWrapperStyle && imageWrapperStyle}>
-            <BackgroundImage
+            <Image
               style={[{ width: null, height: 150 }, imageStyle && imageStyle]}
               source={image}
+              PlaceholderContent={<ActivityIndicator />}
```

Comment:
I'd say remove `PlaceholderContent` just to be consistent with the other components that use Image

Reply:
@iRoachie updated via [f8858bc](https://github.com/react-native-training/react-native-elements/pull/1835/commits/f8858bcd36ad9b2494c0f9a3e0e062e60009cb29).